lib/zendesk_apps_support/package.rb

```diff
@@ -278,6 +279,30 @@ def app_js
       read_file('app.js')
     end
 
+    def location_icons
+      {}.tap do |location_icons|
+        manifest.locations.each do |host, locations_for_host|
+          next unless ['support'].include?(host)
```

Comment:
why do we only do this for support? and if we do still need this guard, could use `ZendeskAppsSupport::Product::SUPPORT.name`?